daemon/cmd/ipam.go

```diff
@@ -309,10 +309,22 @@ func (d *Daemon) allocateDatapathIPs(family types.NodeAddressingFamily, fromK8s,
 func (d *Daemon) allocateHealthIPs() error {
 	bootstrapStats.healthCheck.Start()
 	if option.Config.EnableHealthChecking && option.Config.EnableEndpointHealthChecking {
```

Comment:
Might make sense to reverse the logic here and bail out early to decrease indent?

Reply:
Thank you for your review. I have reversed the logic here.